.github/workflows/tfsec.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: tfsec
+
+on:
+  push:
+    branches: [ "*" ]
```

Comment:
question: do we need this? Isn't it enough to scan the pull requests?

suggestion: if it is needed, we can delete the `tfsec` job from `ci.yml`. If it is not needed, I suggest to move the code to `ci.yml` to replace the `tfsec` job there.